automation_tools/__init__.py

```diff
@@ -1832,68 +1858,68 @@ def sync_capsule_tools_repos_to_upgrade(admin_password=None):
     run(initials+'capsule content synchronize --id {0}'.format(capsule_id))
```

Comment:
Created new Issue https://github.com/SatelliteQE/automation-tools/issues/269  , So that this PR can be merged for now as urgent.